content/applications/inventory_and_mrp/inventory/management/products/strategies.rst

```diff
@@ -1,75 +1,200 @@
+
+When this button is clicked, Odoo will automatically generate a draft :abbr:`PO (purchase order)`/
+:abbr:`MO (manufacturing order)` every time the forecasted stock level falls below the set
+:guilabel:`Min Quantity` of the reordering rule.
+
+On the :guilabel:`Replenishment` page, a reordering rule or manual replenishment can be temporarily
+deactivated for a given period, by clicking the :guilabel:`🔕 (snooze)` icon on the far-right of the
+line.
+
+.. image:: strategies/reordering-rule-snooze-settings.png
+   :align: center
+   :alt: Snooze options to turn off notifications for reordering for a period of time.
 
-Then, click on your product to access the related product form and, on
-the "Inventory submenu", do not forget to select a supplier.
+A :abbr:`PO (purchase order)` or :abbr:`MO (manufacturing order)` created by a manual replenishment
+has a :guilabel:`Replenishment Report` as the source document. A :abbr:`PO (purchase order)` or
+:abbr:`MO (manufacturing order)` created by an automated reordering rule will have the
+:abbr:`SO (sales order(s))` reference number(s) that triggered the rule as the source document.
 
-.. image:: strategies/strategies02.png
+.. image:: strategies/rfq-source-document.png
    :align: center
+   :alt: Quote request list shows which quotes are directly from the replenishment report.
 
-.. tip::
-   Don't forget to select the right product type. A consumable can not be stocked and will thus not
-   be accounted for in the stock valuation.
 
-Make to Order
--------------
+Make to order (MTO)
+===================
 
-The Make to Order configuration is available on your product form
-through your :menuselection:`Inventory --> Inventory control --> Products` (or any
-other module where products are available).
+Since the :abbr:`MTO (make to order)` route is recommended for customized products, the route is
+hidden by default.
 
-On the product form, under Inventory, click on "Make To Order".
+To activate the Make to Order (MTO) route in Odoo:
+ #. Go to :menuselection:`Inventory app --> Configuration  --> Settings`.
+ #. Activate the :guilabel:`Multi-Step Routes` setting, located under the :guilabel:`Warehouse`
+    section, and click :guilabel:`Save`.
+ #. Then, go to :menuselection:`Inventory app --> Configuration  --> Routes`.
+ #. Click on :menuselection:`Filters --> Archived` to show archived routes.
+ #. Select the checkbox next to :guilabel:`Replenish on Order (MTO)`, and click on
+    :menuselection:`Action --> Unarchive`.
 
-.. image:: strategies/strategies03.png
-   :align: center
+.. note::
+   Activating the :guilabel:`Multi-Step Routes` setting also activates :guilabel:`Storage
+   Locations`. If these features aren't applicable to the warehouse, disable these settings after
+   unarchiving the MTO route.
+
+To set a product's procurement route to MTO, go to :menuselection:`Inventory app --> Products -->
+Products`, click on the desired product to open its product form, and click :guilabel:`Edit`.
```

Comment:
```suggestion
To set a product's procurement route to :abbr:`MTO (Make to Order)`, go to :menuselection:`Inventory
app --> Products --> Products`, click on the desired product to open its product form, and click :guilabel:`Edit`.
```
I'd suggest using the abbr throughout for MTO (and other applicable names), just to keep things uniform